setup.cfg

```diff
@@ -1,3 +1,3 @@
 [flake8]
 exclude = tests/*,setup.py
-max-line-length = 160
+max-line-length = 88
```

Comment:
Do you need this config if you're already got a `.flake8` file?

Reply:
Ok, removed setup.cfg in favor of .flake8.